clappr/clappr-ios
# Ping UPA of Apple with now playing

### Goal
Add ``Content identifier`` and ``Playback Progress``  for NowPlaying options.
Add options for the properties above and pass it to the external data on AVPlayerCurrentItem.

### How to test

1. Run all unit tests.
2. Open ViewController.
3. Pass the following options to player:

```swift 
let metadata = [kMetaDataTitle: "Foo",
                        kMetaDataDescription: "Lorem ipsum lorem",
                        kMetaDataContentIdentifier: "Foo v2",
                        kMetaDataDate: Date(),
                        kMetaDataWatchedTime: 1] as [String : Any]
let options = [
            kSourceUrl: "https://devstreaming-cdn.apple.com/videos/streaming/examples/img_bipbop_adv_example_ts/master.m3u8",
            kPosterUrl: "https://cloud.githubusercontent.com/assets/1156242/16349649/54f233e2-3a30-11e6-98e4-42eb5284b730.png",
            kAutoPlay: true,
            kMediaControl: true,
            kMetaData: metadata,
            ] as [String: Any]
```
4. Run the project.
5. On remote control, Swipe down and check if the properties of ``more Info`` is the same that you pass on options.
![nowplaying](https://user-images.githubusercontent.com/2563779/34389488-0fa42d1a-eb21-11e7-9e4d-ac66b9a2b228.png)
